src/main/java/dp/xlsx/XLSXConverter.java

```diff
@@ -20,6 +22,13 @@
 @Component
 public class XLSXConverter {
 
+    private final static Logger LOGGER = LoggerFactory.getLogger(XLSXConverter.class);
+
+    /**
+     * The maximum number of rows to hold in memory
+     **/
+    private static final int MAX_IN_MEMORY_ROWS = 50;
```

Comment:
is it worth making this number configurable (are there any limits on what the library recommends?)

Reply:
I am not sure it's worth it. The defaulted value if you do not specify is 100, I chose 50 on a purely random basis I am not sure it will offer any advantage if it is configurable but if we can identify a reason to make it configurable then I "will make it so".